andela/valhalla-rc
# Deploy to heroku

#### What does this PR do?
Add configurations necessary to deploy app to heroku
#### Description of Task to be completed?
* Pull origin develop to local machine
* Branch out from develop to a new one
* Remove all ^ from package.json except **jest** configuration
* Push branch to develop
* Login to heroku dashboard and create a new app
* Add mlab mongodb add-on(you may need to include billing info), papertrail(optional) for comprehensive log
#### How should this be manually tested?
  * Open created heroku app after setup is complete to ensure the right content is served
#### Any background context you want to provide?
* Link [here](https://medium.com/@oreagunbiade/reactioncommerce-heroku-deploy-guide-f5672e242863) to a comprehensive guide on deployment written by an Andelan
* At the end of the setup, the following config vars will be in the app settings on heroku dashboard

            BUILDPACK_CLEAR_CACHE=
            BUILDPACK_PRELAUNCH_METEOR=
            MONGODB_URI=
            PAPERTRAIL_API_TOKEN(available if papertrail add-on is added)
            ROOT_URL=
            TOOL_NODE_FLAGS=--max-old-space-size=4096

            All url in the config vars will be in https instead of http
* When app is successfully created, pipelines can then be created.
#### What are the relevant pivotal tracker stories?
#158911766
#### Screenshots (if appropriate)
#### Questions:
